zerver/openapi/zulip.yaml

```diff
@@ -1864,6 +1864,45 @@ paths:
                                     [["Denmark", "topic", 1594825442]],
                                   "id": 0,
                                 }
+                            - type: object
+                              description: |
+                                Event sent to a user's clients when that user's set of
+                                configured user-topic relationships have changed.
+                              properties:
+                                id:
+                                  $ref: "#/components/schemas/EventIdSchema"
+                                type:
+                                  allOf:
+                                    - $ref: "#/components/schemas/EventTypeSchema"
+                                    - enum:
+                                        - user_topic
+                                stream_id:
+                                  type: integer
+                                  description: |
+                                    The ID of the stream to which the topic belongs.
+                                topic_name:
+                                  type: string
+                                  description: |
+                                    The name of the topic.
+                                last_updated:
+                                  type: integer
+                                  description: |
+                                    An integer UNIX timestamp representing when the user-topic
+                                    relationship was changed.
+                                visibility_policy:
+                                  type: integer
+                                  description: |
+                                    The type of visibility policy followed by the user-topic pair.
```

Comment:
This should document the value values, which for now are just:
* 0 => "None", used in events to indicate this topic no longer has any user_topic configuration.
* 1 => "Muted", used to record muted topics.